src/components/pagination.js

```diff
@@ -8,12 +8,12 @@ import { DropdownAutoCenter } from '..'
 const Pagination = ({ classes, itemsLength, onChangePage, onChangeRowsPerPage, initialPage, pageSize, showPage, firstLast, counter, rowsPerPage, hideRowsPerPage }) => {
   const paginationClasses = Object.freeze({
     container: `pagination__main-container flex items-center text-xs tracking-md leading-1.33 bg-secondary-50
-      ${classes.container && classes.container}`,
+      ${classes.container || ''}`,
```

Comment:
@amy83762100 need to move all these classes before any other styling, after the class names, for all classes keys.